Functions/Functions/User/AddUserOrganisationFunction.cs

```diff
@@ -45,7 +45,7 @@ public AddUserOrganisationFunction(RulesDbContext dbContext, IApiAuthorization a
             string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
             data = JsonConvert.DeserializeObject<User>(requestBody);
 
-            bool isNull = string.IsNullOrEmpty(data?.OrganisationId) || string.IsNullOrEmpty(data?.UserId);
+            bool isNull = data?.OrganisationId == null || string.IsNullOrEmpty(data?.UserId);

```

Comment:
Please change this to do everything in the if statement as per my comment in the last review